src/components/countries/Countries.js

```diff
@@ -0,0 +1,61 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import getCountries from '../../redux/thunk/countries';
+import Header from '../Header';
+import CountryItem from './CountryItem';
+
+const Countries = () => {
```

Comment:
Kindly take a look at the project screenshot below
- [ ] Your Home page (**Countries** component) is missing a filtering mechanism, like a search functionality, by which users can filter countries using a parameter. Kindly add this.

  ![shah-search](https://user-images.githubusercontent.com/36542986/146970638-3ceaea37-3415-4128-aac9-b79cb2a58e46.jpg)